src/error/metadata/suggestion.rs

```diff
@@ -220,6 +224,9 @@ UpgradePlan => "Rover has likely reached rate limits while running graph or subg
                 format!("The subgraph {} does not exist for {}. You cannot add a subgraph to a supergraph without a routing URL.
                 Try re-running this command with a `--routing-url` argument.", subgraph_name, Style::Link.paint(graph_ref.to_string()))
             }
+            LinkPersistedQueryList { graph_ref, frontend_url_root } => {
+                format!("Link a persisted query list to {graph_ref} by heading to {frontend_url_root}/graph/{id}/{variant}/persisted-queries", id = graph_ref.name, variant = graph_ref.variant)
```

Comment:
`variant` is wrong here — recall that PQs are at the graph level.

```suggestion
                format!("Link a persisted query list to {graph_ref} by heading to {frontend_url_root}/graph/{id}/persisted-queries", id = graph_ref.name)
```

(or just pass the ID into this object rather than the full graph ref)